src/lib/services/amplitude.tsx

```diff
@@ -209,10 +218,18 @@ export const AmpTrackCopier = (section: Option<string>, type: string) =>
 
 export const AmpTrackExpand = (
   action: "expand" | "collapse",
-  component: "assets" | "json" | "permission_address" | "event_box",
+  component:
+    | "assets"
+    | "json"
+    | "permission_address"
+    | "event_box"
+    | { type: "unsupported_pool"; id: number },
   section: Option<string>
 ) => track(AmpEvent.USE_EXPAND, { action, component, section });
```

Comment:
1. receive props as object 
```ts 
AmpTrackExpand = ({action, component,...}: interface) => {}
```
2. just | "unsupported_pool" and receive another prop (info: object)